### DISCUSSION TITLE
Add indication that it is directly possible to *copy* the graph ready to be downloaded

### DISCUSSION THREAD

**Discussion by marcopiani** [owid/owid-grapher#1465] [Ideas]
Hi,

I think that most users of the Grapher may not realize that, once one chooses to download the graph (first image), the *full-size* graph (rather than the smaller rendered graph) can be immediately copied (second image) to be pasted somewhere else without the need to actually download it locally.

Making this possibility better known might be helpful as:
- it means the user can choose not to "clog" any download folder;
- knowing this may encourage more people to use the higher-quality downloadable graph, rather than a screen grab.

I think one could advertise this possibility by simply adding some text at the step where one chooses the download option, without requiring much more work on your side.

Thank you for the consideration you might give to this issue, and, as always, for all your work.

Marco

![image](https://user-images.githubusercontent.com/18381804/170147027-09757d95-6699-4c2b-91fe-a181512d0bed.png)

![image](https://user-images.githubusercontent.com/18381804/170147127-7e338b52-bffa-47a6-ba4f-83af89d7e240.png)

**Comment by marcelgerber** [comment]
Hi Marco,

great that you found out about this, it is indeed very helpful.
Just last week a colleague of us was suggesting something similar, so we created #1441 to track this. There we proposed adding a "Copy chart image" button directly, either to the Download tab or directly to the share menu. We believe this to be the better and even nicer option :)

However, as you will know by now, there are always many more important things to do than this "nice to have" feature, so it might take a while until we can get to it.

Thank you for the suggestion!

**Reply by marcopiani** [reply]
Great to know you are already on it!

I have been using this "feature" for very long (>1year?). I just thought it was worth spreading the word about the "trick". Good that you are planning to make it even more transparent with a "copy" button/feature! 

Note also the idea in https://github.com/owid/owid-grapher/discussions/1464 if you are making any changes to the download/copy feature.

Thanks!

Marco
